docs/hub/datasets-duckdb-cli-sql.md

```diff
@@ -0,0 +1,159 @@
+# Perform SQL operations
+
+Performing SQL operations with DuckDB opens up a world of possibilities for querying datasets efficiently. Let's dive into some examples showcasing the power of DuckDB functions.
+
+For our demonstration, we'll explore a fascinating dataset. The [MMLU](https://huggingface.co/datasets/cais/mmlu) dataset is a multitask test containing multiple-choice questions spanning various knowledge domains.
+
+To preview the dataset, let's select a sample of 3 rows:
+
+```bash
+FROM 'hf://datasets/cais/mmlu/all/test-*.parquet' USING SAMPLE 3;
+
+┌──────────────────────┬──────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬────────┐
+│       question       │       subject        │                                                                         choices                                                                          │ answer │
+│       varchar        │       varchar        │                                                                        varchar[]                                                                         │ int64  │
+├──────────────────────┼──────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┼────────┤
+│ Dr. Harry Holliday…  │ professional_psych…  │ [discuss his vacation plans with his current clients ahead of time so that they know he’ll be unavailable during that time., give his clients a phone …  │      2 │
+│ A resident of a st…  │ professional_law     │ [The resident would succeed, because the logging company's selling of the timber would entitle the resident to re-enter and terminate the grant to the…  │      2 │
```

Comment:
maybe we have to do something to improve the markdown rendering which is broken by the `'` in `logging company's selling`